pom.xml

```diff
@@ -400,6 +400,21 @@
       <artifactId>checker-qual</artifactId>
       <version>${checkerframework.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.maven.doxia</groupId>
+      <artifactId>doxia-core</artifactId>
+      <version>1.12.0</version>
+    </dependency>
+    <dependency>
+        <groupId>org.apache.maven.doxia</groupId>
+        <artifactId>doxia-module-xdoc</artifactId>
+        <version>1.12.0</version>
+    </dependency>
+    <dependency>
+      <groupId>com.puppycrawl.tools</groupId>
+      <artifactId>checkstyle</artifactId>
+      <version>10.12.2-SNAPSHOT</version>
+    </dependency>
```

Comment:
@rnveach Is this what you were talking about?

Reply:
Thats how I got it to work locally, but as I mentioned, talk to the others if there is a better way as this will require an install beforehand. If at the least, version should be replaced with a variable for the current defined version (Like `${project.version}`).

Reply:
Suuuper strange. I just got a new laptop today. Ran it twice and the second time it worked without the `checkstyle` dependency.

Reply:
If you don't run clean, doxia uses the previous run as a cached version.

Reply:
You are right. uhhh, I thought it fixed itself or something.